c/termination-crafted/Cairo_step2_true-no-overflow_false-termination_true-valid-memsafety.c

```diff
@@ -13,7 +13,7 @@ int main()
     int x;
     x = __VERIFIER_nondet_int();
 	if (x > 0) {
-	    while (x != 0) {
+	    while (x != 0 && x!= -1) {
```

Comment:
This will always terminate now. Change verdict to `true-termination`.

Reply:
Is there a way to remove the overflow without changing the termination?

Reply:
One could change the type of x to unsigned int, but that would have the disadvantage that the absence of an overflow is (too) trivial (wrapping around for unsigned integer types is well defined).
Both approaches seem reasonable. If there are no objections, I will
- change the label of the current file to `true-termination`
- add a version with `unsigned int x;`, changed initialization of x to  `x = __VERIFIER_nondet_uint();` (and the while-loop only checks for x!=0), with label `false-termination`